.golangci.yml

```diff
@@ -39,6 +38,10 @@ issues:
     linters:
     - errcheck
     - gosec
+  - path: pkg/reconciler/pipelinerun/pipelinerun_test\.go
```

Comment:
Was wondering if this shall be `test/pipelinerun_test\.go` ?

Reply:
Good catch Jerome! I was blinded by my previous pass and just saw "why is `unused` still failing on `pipelinerun_test.go` when I told it to skip that file!?" -- which caused me to not notice that it was the same filename in a different path. Fixed, and now the linter is passing (in my local run)!